README.md

```diff
@@ -175,7 +175,4 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-A copy of the license is available in the repository's [LICENSE](./license.txt) file.
-
-[](Esri Tags: ArcGIS Web Mapping Leaflet AppBuilder Hackerlabs Labs JavaScript)
-[](Esri Language: JavaScript)
+A copy of the license is available in the repository's [LICENSE](./LICENSE) file.
```

Comment:
i changed to all caps and dropped the file extension in this pull request.

0  license.txt → LICENSE

you can test yourself whether links in proposed markdown are functioning by clicking the `view` button to display the entire file.

https://github.com/Esri/geodev-hackerlabs/blob/decc3eb84746622e06b1aa2dbce489f325f5035c/README.md#licensing